portfolio/src/main/webapp/script.js

```diff
@@ -50,8 +50,48 @@ function showPicture(){
   imageContainer.appendChild(imgElement);    
 }
 
-function sayHello(){
-  fetch('/data').then(response => response.text()).then((statement) => {
-    document.getElementById('server-container').innerText = statement;
+function getDatastoreComments(){
+  fetch('/data').then(response => response.json()).then((comments) => {
+    const serverContainer = document.getElementById('server-container');
+    comments.forEach((comment) => {
+      serverContainer.appendChild(createCommentBox(comment));
```

Comment:
Seems each time when I click "show comments", it will append the same comments, you'll see accumulated duplicate comments, maybe try to clean the existing child first and then append.